docs/UserGuide.md

```diff
@@ -33,6 +33,8 @@ title: User Guide
   [3.4.2. Deleting item in profile: `delete me`](#deleting-item-in-profile-delete-me) <br />
   [3.4.3. Editing item in profile: `edit me`](#editing-item-in-profile-edit-me) <br />
   [3.4.4. Viewing item in profile: `view me`](#viewing-item-in-profile-view-me) <br />
+  [3.1.5. Finding items in profile: `find me`](#finding-items-in-profile-find-me) <br />
```

Comment:
the numbering is wrong, but I have updated this in my current PR for its ToC

Reply:
Okay but my find header is different, I had a different header from shawn but he hasnt gotten back to me about changing yet